pyiode/iode_database/lists_database.pyx

```diff
@@ -158,16 +156,13 @@ cdef class Lists(_AbstractDatabase):
         ['COMMA_LST', 'MIX_LST', 'SEMICOLON_LST', 'TAB_LST', 'WHITESPACE_LST']
         >>> lists["SEMICOLON_LST"]
         'A;B;C;D;E;F'
-        >>> split_list(lists["SEMICOLON_LST"])
-        ['A', 'B', 'C', 'D', 'E', 'F']
-        >>> # note: leading and trailing whitespaces are automatically stripped 
-        >>> #       when copied into the Lists database
-        >>> mix_lst
-        '  A;B,C D    E;F  '
+        >>> # note: when added or updated, an IODE list is normalized.
+        >>> #       - leading and trailing whitespaces are stripped
+        >>> #       - items of the IODE list are separated by a semicolon
+        >>> lists["COMMA_LST"]
+        'A;B;C;D;E;F'
```

Comment:
Now that I think of it, I **wonder** if it wouldn't it be more convenient if individual lists came out as Python lists instead of strings. It surely would feel more Pythonic but I honestly don't know if it would be less or more convenient to work with them that way. 

I am pretty sure users will want the possibility to define them using a string (because that's how they have been doing it and it is much shorter to type) but when they get it back our of the database, what do they do with it? If they want to iterate or modify it, I think a Python list would be more convenient, because they do not need to "unpack" then "repack" it.

Accepting a Python list (in addition to a string) as value in `__setitem__`/_set_object might be some kind of middle ground, but then we drift again to the side of "we accept anything and everything in our APIs" 😉 .

Reply:
made 2 separate issues : #590 (getitem returns a Python list) and #591 (setitem accept a Python list)